### PR TITLE
Fixes metrics response code and handles uris same as Micrometer WebMVC

### PR DESCRIPTION
**Pull request by zeagord**
This is to ensure dimensions don't explode on variables

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
It isn't obvious to me why we would intentionally not track UI latency... Mind elaborating a bit more? I might just not understand.

File: zipkin-autoconfigure/metrics-prometheus/src/main/java/zipkin/autoconfigure/prometheus/ZipkinPrometheusMetricsAutoConfiguration.java [hunk 1]
```diff
@@ -75,11 +74,22 @@ UndertowDeploymentInfoCustomizer httpRequestDurationCustomizer() {
...
+        , Tag.of("path", cleanPathTemplate(exchange.getRelativePath()))
         , Tag.of("status", Integer.toString(exchange.getStatusCode()))
       );
     }
 
+    private String cleanPathTemplate(String path){
+      path = path.replaceAll("//", "/");
+      if (path.contains("/api/v1/trace"))
+        return "/api/v1/trace/{traceId}";
+      if (path.contains("/zipkin/traces/"))
```
Comment (codefromthecrypt) [inline zipkin-autoconfigure/metrics-prometheus/src/main/java/zipkin/autoconfigure/prometheus/ZipkinPrometheusMetricsAutoConfiguration.java]: ahh this must be for crossroads

**Comment by codefromthecrypt** [comment]
thanks.. I'll pull locally also split the logging thing from the other thing

**Comment by codefromthecrypt** [comment]
we can't remove the log4j thing to zipkin-autoconfigure as it breaks kafka when run as an exec jar..

```
[INFO] 2018-04-29 10:43:46.809  WARN [/] 50622 --- [           main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'kafka' defined in class path resource [zipkin/autoconfigure/collector/kafka/ZipkinKafkaCollectorAutoConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [zipkin.collector.kafka.KafkaCollector]: Factory method 'kafka' threw exception; nested exception is java.lang.NoClassDefFoundError: org/apache/log4j/Logger
```

**Comment by codefromthecrypt** [comment]
moved log4j stuff to a separate PR https://github.com/openzipkin/zipkin/pull/2032

**Review by shakuzen — COMMENTED** [review]
After the fact, but LGTM

**Comment by codefromthecrypt** [comment]
reviews before during and after the fact greatly appreciated, especially by
users like yourself!

**Pull request merged**